specification/dimensions/usageunit.md

```diff
@@ -0,0 +1,51 @@
+# Usage Unit
+
+Usage Unit refers to a unit of measurement for the consumption or usage of resources or services. The Usage Unit for a particular line item may differ from the published Pricing Unit when providers use different units, unit increments, or dimensions for calculating cost.  This unit is commonly used when auditing or reconciling consumption or billing data.
+
+The UsageUnit column MUST be present and MUST NOT be null or empty. UsageUnit is of type String, and UsageUnit MUST be composed of the list of recommended units or unit abbreviations if it is possible to be expressed with the units listed in "Recommended Unit Set" below. If the UsageUnit value is a composite value made from combinations of one or more units, each component MUST also align with the set of recommended units if possible.  For example, a composite unit of "GB/Month" is allowed.
```

Comment:
If we are saying it MUST if possible - wouldn't it be more concise to say SHOULD?

Reply:
Agreed, I tried to preserve the language of the original, but I agree that is an odd contradiction, you "MUST" do this if it's possible.  I think we need to change it into something more along the lines of:

Reply:
@MichaelArkoosh-vega - see the PR I just opened against this branch.